receiver/splunkenterprisereceiver/testdata/scraper/expected.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this expected?

Reply:
It was not expected. I have fixed the broken test